apps/src/sites/studio/pages/levels/_content.js

```diff
@@ -1,19 +1,47 @@
 import $ from 'jquery';
+import assetUrl from '@cdo/apps/code-studio/assetUrl';
 import React from 'react';
 import ReactDom from 'react-dom';
-
 import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import {convertXmlToBlockly} from '@cdo/apps/templates/instructions/utils';
+import commonBlocks from '@cdo/apps/blocksCommon';
 
 $(document).ready(() => {
+  // Content levels don't have a particular 'app' associated with them so make
+  // available all the different app specific Blockly blocks as they are needed
+  // by level creators. This will enable them to write Blockly XML and render
+  // the blocks directly in the level text.
+  Blockly.assetUrl = assetUrl;
+  Blockly.Css.inject(document);
+  // Install the common Blockly blocks
+  commonBlocks.install(window.Blockly, {});
+  // Install the custom CDO blocks for the following level/app types.
+  var options = [{skin: {id: 'birds'}, app: 'maze'}, {skin: {}, app: 'turtle'}];
```

Comment:
note this is inconsistent with the above comment, which claims that we are making _all_ the apps available.

I don't think I'm in favor of this approach; in particular, the existing app-specific locale files are not intended to be used together and there is no guarantee that they won't have conflicts. I'd recommend that we install only the common blockly blocks for now, and as follow-up work look into either adding an app association to content levels or unifying the app locales in general.

Reply:
s/locale/block

Reply:
From our conversation offline, I have created TODO's and a follow up JIRA task which we will get prioritized for Jan 2020.
https://codedotorg.atlassian.net/browse/FND-972

Reply:
I talked with Bryan and found out he is mostly interested in being able to render Turtle Blockly blocks. So I have removed the second level type "maze" from the blocks imported. This way we don't have to do cleanup in January and we will still do the dynamic level type loading later in H1.